spack/packages/dla-future/package.py

```diff
@@ -43,7 +44,8 @@ class DlaFuture(CMakePackage, CudaPackage, ROCmPackage):
     depends_on("umpire+rocm~shared", when="+rocm")
     depends_on("umpire@4.1.0:")
 
-    depends_on("pika@0.16:")
+    depends_on("pika@0.15.1:", when="@0.1.0")
```

Comment:
Maybe
```suggestion
    depends_on("pika@0.15.1:", when="@0.1")
```
? Definitely not a blocker.